docs/python/tutorial-django.md

```diff
@@ -857,19 +855,24 @@ With your models in place and the database migrated, you can store and retrieve
 1. In the app's `urls.py`, import the data model:
 
     ```python
-    from .models import LogMessage
+    from hello.models import LogMessage
+    ```
+
+1. Also in `urls.py`, make a variable for the new view, which retrieves the five most recent `LogMessage` objects in descending order (meaning that it queries the database), and then provides a name for the data in the template context (`message_list`), and identifies the template to use:
+
+    ```python
+    home_list_view = views.HomeListView.as_view(
+        queryset=LogMessage.objects.order_by("-log_date")[:5],  # :5 limits the results to the five most recent
+        context_object_name="message_list",
+        template_name="hello/home.html",
+    )
     ```
 
-1. Also in `urls.py`, replace the path for the home page with the code below, which retrieves the five most recent `LogMessage` objects in descending order (meaning that it queries the database), and then provides a name for the data in the template context (`message_list`), and identifies the template to use:
+1. Make one more change in `urls.py`, and replace the path for the home page with the code below to use our newly declared view.
```

Comment:
Replace the text for this item with just "In `urls.py`, modify the path to the home page to use the `home_list_view` variable:"